scheduler/scheduler-api/build.gradle

```diff
@@ -8,6 +8,8 @@ dependencies {
     compile 'commons-io:commons-io:2.5'
     compile group: 'org.springframework', name: 'spring-expression', version: '4.2.5.RELEASE'
 
+    compile 'joda-time:joda-time:2.9.1'
```

Comment:
Could we only use Java Date instead of introducing a library?
If not, I see that joda is only used in tests, so would it be possible to have this dependency in tests only, e.g. with `testCompile`?

Reply:
Agree to change to testCompile. 
However, this library is already used in scheduer-server for compile.